.github/odbc/install-mariadb-driver.sh

```diff
@@ -0,0 +1,9 @@
+sudo apt-get install -y cmake
```

Comment:
Can you please add a comment here to remind us of why we're not just using `apt-get`?

Reply:
Done: Added a note pointing to this PR.  Expanded the description of this PR with detail on which tests are failing and how with the canned mysql driver.